### ISSUE TITLE
the performance of ncf in hr@10 didn't achieve 0.7

### ISSUE THREAD

**Issue by LiuHao-THU** [yihong-chen/neural-collaborative-filtering#12]
the results of hr@10 using ncf model is blow 0.7,however in paper "neural collaborative filtering" is over 0.72 using factor 32

**Comment by yihong-chen** [comment]
Thank you for reporting the issue. Could you add the exact hyperparameters settings for this ? NCF is quite sensitive to hyperparameters. I didn't spend a lot of time tuning them. I was suspecting all the weight decay(regularization) and finetuning. Would you like to help with this and report the results here?

**Comment by LiuHao-THU** [comment]
here is a link for NCF BPR and NFM
https://github.com/guoyang9/NCF
https://github.com/guoyang9/BPR-pytorch
https://github.com/guoyang9/NFM-pyorch
the scores are exactly the same even higher than origin papers!

**Comment by yihong-chen** [comment]
Nice points! I checked the repos. I didn't do the weight initialization. Maybe that's the reason. You could use those repos if you want.

**Comment by yihong-chen** [comment]
@LiuHao-THU Could you elaborate a bit on `factor` ? Is it latent factor or predictive factor?
